storage/remote/client_test.go

```diff
@@ -65,11 +65,15 @@ func TestStoreHTTPErrorHandling(t *testing.T) {
 		if err != nil {
 			t.Fatal(err)
 		}
-
-		c, err := NewClient(0, &ClientConfig{
+		conf := &ClientConfig{
 			URL:     &config_util.URL{URL: serverURL},
 			Timeout: model.Duration(time.Second),
-		})
+		}
+		hash, err := toHash(conf)
+		if err != nil {
+			t.Fatal(err)
+		}
+		c, err := NewClient(hash, conf)
```

Comment:
Since this is just a test could a hardcoded string as a name work?

Reply:
not sure what you mean

Reply:
I am not sure anymore...